clang/test/Modules/reproducer-with-module-dependencies.c

```diff
@@ -76,3 +83,8 @@ CHECK: -ivfsoverlay "reproducer.cache/vfs/vfs.yaml"
 CHECK: "-ivfsoverlay" "{{.*}}/existing.yaml"
 CHECK: MACRO=\$foo
 CHECK: "-dependency-file" "reproducer.cache/deps.d"
+
+//--- cas-script-expectations.txt
+CHECK: -fcas-path "reproducer.cache/cas"
+CHECK: "-fcas-include-tree" "llvmcas://
```

Comment:
I don't know if it is worth it (or common) to check the presence of an include tree in a generated CAS.

Reply:
For swift, I just find a way to execute the reproducer.